### PR TITLE
removed blueprint library

### PR DESCRIPTION
**Pull request by tonyxuan2021**
- removed blueprint library only, and made sure server can start without error. 
- next step is to switch database from sqlite to PostGresql

### CODE REVIEW

**Review by chunloy — COMMENTED** [review]
Looks good to me, but I do see some unused code. Maybe a separate PR to address this?

File: app.py [hunk 1]
```diff
@@ -1,25 +1,78 @@
...
 with app.app_context():
     db.create_all()
 
 
+@app.route("/driverLogin", methods=["POST"])
+def driverlogin():
+    data = json.loads(request.get_data())
+    if not data["username"] or not data["password"]:
+        return "Please enter the required fields", 400
+    driver_login = Users.query.filter_by(username=data["username"]).first()
```
Comment (yhus) [inline app.py:25]: Move business logic to separate function

File: app.py [hunk 2]
```diff
@@ -1,25 +1,78 @@
...
+    return encoded_jwt
+    # return data
+
+
+@app.route("/riderLogin", methods=["POST"])
+def riderLogin():
+    data = json.loads(request.get_data())
+    if not data["username"] or not data["password"]:
+        return "Please enter the required fields", 400
+    rider_login = Riders.query.filter_by(username=data["username"]).first()
```
Comment (yhus) [inline app.py:43]: Same here, move business logic out

File: app.py [hunk 3]
```diff
@@ -1,25 +1,78 @@
...
+    # auth_token = request.headers.get("Authorization").split(" ")[1]
+    auth_token = request.headers.get("Authorization")
+
+    try:
+        driver_entered_data = jwt.decode(auth_token, SECRET_KEY, algorithms="HS256")
+        current_logged_in_user = Users.query.filter_by(
+            username=driver_entered_data["username"]
+        ).first()
+
+        print(current_logged_in_user.username)
```
Comment (yhus) [inline app.py:66]: Remove? Is this needed?

File: app.py [hunk 4]
```diff
@@ -1,25 +1,78 @@
...
+
+        return "some route info", 200
+
+        # return jsonify(current_logged_in_user)
```
Comment (yhus) [inline app.py:70]: Remove comments

**Comment by tonyxuan2021** [comment]
>

File: app.py [hunk 5]
```diff
@@ -1,25 +1,78 @@
...
+    )
+    # print(encoded_jwt)
+    return encoded_jwt
+    # return data
+
+
+@app.route("/riderLogin", methods=["POST"])
+def riderLogin():
+    data = json.loads(request.get_data())
+    if not data["username"] or not data["password"]:
```
Comment (yhus) [inline app.py:41]: This is duplicated code from the method above. Can you refractor out to separate helper function and call it from whichever endpoints need to handle logins?

File: app.py [hunk 6]
```diff
@@ -1,25 +1,78 @@
...
+def driveroutes():
+    if not request.headers.get("Authorization"):
+        return "Please enter the required fields", 401
+
+    # auth_token = request.headers.get("Authorization").split(" ")[1]
+    auth_token = request.headers.get("Authorization")
+
+    try:
+        driver_entered_data = jwt.decode(auth_token, SECRET_KEY, algorithms="HS256")
+        current_logged_in_user = Users.query.filter_by(
```
Comment (yhus) [inline app.py:62]: Same here, move business logic out

**Review by yhus — CHANGES_REQUESTED** [review]

File: app.py (hunk 1 above)
Comment (tonyxuan2021) [reply app.py:25]: done.

File: app.py (hunk 2 above)
Comment (tonyxuan2021) [reply app.py:43]: done.

File: app.py (hunk 3 above)
Comment (tonyxuan2021) [reply app.py:66]: unused comment has been removed.

File: app.py (hunk 4 above)
Comment (tonyxuan2021) [reply app.py:70]: removed.

File: app.py (hunk 5 above)
Comment (tonyxuan2021) [reply app.py:41]: Thank you. I have created a helper function to handle duplicate code.

File: app.py (hunk 6 above)
Comment (tonyxuan2021) [reply app.py:62]: Done.

**Pull request merged**